debian/ideascube.postinst

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+
+set -e
+
+case "$1" in
+    configure)
+        if ! getent passwd "$DPKG_MAINTSCRIPT_PACKAGE" > /dev/null 2>&1 ; then
+            adduser --quiet \
+                    --system \
+                    --no-create-home \
+                    --group \
```

Comment:
Quoting the `adduser(8)` : 

> **--group**
> When  combined  with --system, a group with the same name and ID as the system user is created.  If not combined with --system, a group  with  the  given  name  is  created.  This is the default action if the program is invoked as addgroup.

We are running `adduser` with the `--system` option, therefore no, we do not need to specify the group name.

Note that I did test the command. :smiley: